seacrowd/sea_datasets/mlqa/mlqa.py

```diff
@@ -0,0 +1,244 @@
+    German, Spanish, Hindi, Vietnamese and Simplified Chinese. MLQA is highly parallel, with QA instances parallel between
+    4 different languages on average.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    subsets = [
+        "mlqa-translate-test.vi",
+        "mlqa-translate-train.vi",
+        "mlqa.vi.ar",
+        "mlqa.vi.de",
+        "mlqa.vi.zh",
+        "mlqa.vi.en",
+        "mlqa.vi.es",
+        "mlqa.vi.hi",
+        "mlqa.vi.vi",
+        "mlqa.ar.vi",
+        "mlqa.de.vi",
+        "mlqa.zh.vi",
+        "mlqa.en.vi",
+        "mlqa.es.vi",
+        "mlqa.hi.vi",
+    ]
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="{sub}_source".format(sub=subset),
+            version=datasets.Version(_SOURCE_VERSION),
+            description="{sub} source schema".format(sub=subset),
+            schema="source",
+            subset_id="{sub}".format(sub=subset),
+        )
+        for subset in subsets
+    ] + [
+        SEACrowdConfig(
+            name="{sub}_seacrowd_qa".format(sub=subset),
+            version=datasets.Version(_SEACROWD_VERSION),
+            description="{sub} SEACrowd schema".format(sub=subset),
+            schema="seacrowd_qa",
+            subset_id="{sub}".format(sub=subset),
+        )
+        for subset in subsets
+    ]
+
+    DEFAULT_CONFIG_NAME = "mlqa.vi.vi_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {"context": datasets.Value("string"), "question": datasets.Value("string"), "answers": datasets.Features({"answer_start": [datasets.Value("int64")], "text": [datasets.Value("string")]}), "id": datasets.Value("string")}
+            )
+        elif self.config.schema == "seacrowd_qa":
+            features = schemas.qa_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        name_split = self.config.name.split("_")
+        url = ""
+        data_path = ""
+
+        if name_split[0].startswith("mlqa-translate-train"):
+            config_name, lang = name_split[0].split(".")
+            url = _URL + _TRANSLATE_TRAIN_URL
+            data_path = dl_manager.download(url)
+            return [
+                datasets.SplitGenerator(
+                    name=datasets.Split.TRAIN,
+                    # Whatever you put in gen_kwargs will be passed to _generate_examples
+                    gen_kwargs={
+                        "filepath": f"{config_name}.{lang}/{lang}_squad-translate-train-train-v1.1.json",
```

Comment:
the expected folder is `mlqa-translate-train` only so the lang is not needed in the folder part
```suggestion
                        "filepath": f"{config_name}/{lang}_squad-translate-train-train-v1.1.json",
```